Plan/common/src/main/resources/assets/plan/web/css/style.css

```diff
@@ -759,121 +759,126 @@ div.scrollbar {
     color: #fff;
 }
 
-.bg-purple {
+.bg-purple, body.theme-purple .fc-toolbar-chunk .btn.btn-primary {
     background-color: #9C27B0;
     color: #fff;
 }
 
-.bg-deep-purple {
+.bg-deep-purple, body.theme-deep-purple .fc-toolbar-chunk .btn.btn-primary {
     background-color: #673AB7;
     color: #fff;
 }
 
-.bg-indigo {
+.bg-indigo, body.theme-indigo .fc-toolbar-chunk .btn.btn-primary {
     background-color: #3F51B5;
     color: #fff;
 }
 
-.bg-blue {
+.bg-blue, body.theme-blue .fc-toolbar-chunk .btn.btn-primary {
     background-color: #2196F3;
     color: #fff;
 }
 
-.bg-light-blue {
+.bg-light-blue, body.theme-light-blue .fc-toolbar-chunk .btn.btn-primary {
     background-color: #03A9F4;
     color: #fff;
 }
 
-.bg-cyan {
+.bg-cyan, body.theme-cyan .fc-toolbar-chunk .btn.btn-primary {
     background-color: #00BCD4;
     color: #fff;
 }
 
-.bg-teal {
+.bg-teal, body.theme-teal .fc-toolbar-chunk .btn.btn-primary {
     background-color: #009688;
     color: #fff;
 }
 
-.bg-green {
+.bg-green, body.theme-green .fc-toolbar-chunk .btn.btn-primary {
     background-color: #4CAF50;
     color: #fff;
 }
 
-.bg-light-green {
+.bg-light-green, body.theme-light-green .fc-toolbar-chunk .btn.btn-primary {
     background-color: #8BC34A;
     color: #fff;
 }
 
-.bg-lime {
+.bg-lime , body.theme-lime .fc-toolbar-chunk .btn.btn-primary{
     background-color: #CDDC39;
     color: #fff;
 }
 
-.bg-yellow {
+.bg-yellow, body.theme-yellow .fc-toolbar-chunk .btn.btn-primary {
     background-color: #ffe821;
     color: #fff;
 }
 
-.bg-amber {
+.bg-amber, body.theme-amber .fc-toolbar-chunk .btn.btn-primary {
     background-color: #FFC107;
     color: #fff;
 }
 
-.bg-orange {
+.bg-orange, body.theme-orange .fc-toolbar-chunk .btn.btn-primary {
     background-color: #FF9800;
     color: #fff;
 }
 
-.bg-deep-orange {
+.bg-deep-orange, body.theme-deep-orange .fc-toolbar-chunk .btn.btn-primary {
     background-color: #FF5722;
     color: #fff;
 }
 
-.bg-brown {
+.bg-brown, body.theme-brown .fc-toolbar-chunk .btn.btn-primary {
     background-color: #795548;
     color: #fff;
 }
 
-.bg-grey {
+.bg-grey, body.theme-grey .fc-toolbar-chunk .btn.btn-primary {
     background-color: #9E9E9E;
     color: #fff;
 }
 
-.bg-blue-grey {
+.bg-blue-grey, body.theme-blue-grey .fc-toolbar-chunk .btn.btn-primary {
     background-color: #607D8B;
     color: #fff;
 }
 
-.bg-black {
+.bg-black, body.theme-black .fc-toolbar-chunk .btn.btn-primary {
     background-color: #555555;
     color: #fff;
 }
 
-.bg-white {
+.bg-white, body.theme-white .fc-toolbar-chunk .btn.btn-primary {
     background-color: #ffffff;
     color: #333;
 }
 
-.bg-plan {
+.bg-plan, body.theme-plan .fc-toolbar-chunk .btn.btn-primary {
     background-color: #368F17;
     color: #fff;
 }
 
-.bg-night {
+.bg-night, body.theme-night .fc-toolbar-chunk .btn.btn-primary {
     background-color: #44475a;
     color: #eee8d5;
 }
 
+.fc-toolbar-chunk .btn.btn-primary {
+    outline: none !important;
```

Comment:
Sidenote: Setting outline to none disables visual aid for users that navigate the site using tab, such as those with low vision and screen readers.
I'm just noting this down and not requesting changes because the Plan website is not really accessibility friendly regardless.
I learned about a11y this summer so the plugin has not gone through treatment yet.